### PR TITLE
schema client docker & ci update

### PR DESCRIPTION
**Pull request by choks87**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,11 +15,11 @@ fix-code-style:
...
 
 code-style:
-	${PHPCS} --report-full --report-gitblame --standard=PSR12 ./src
+	mkdir -p build/logs/phpcs
+	${PHPCS}
 
 coverage:
-	mkdir -p build/logs/phpunit
-	${PHPDBG} # && ./vendor/bin/coverage-check build/logs/phpunit/coverage/coverage.xml 100
+	${PHPUNIT} # && ${COVCHK} build/logs/phpunit/coverage/coverage.xml 100
```
Comment (nick-zh) [inline Makefile]:
```suggestion
	${PHPUNIT} && ${COVCHK} build/logs/phpunit/coverage/coverage.xml 100
```

File: Makefile [hunk 2]
```diff
@@ -2,11 +2,11 @@
 .DEFAULT_GOAL := test
 
 PHPUNIT =  ./vendor/bin/phpunit -c ./phpunit.xml
-PHPDBG =  phpdbg -qrr ./vendor/bin/phpunit -c ./phpunit.xml
-PHPSTAN  = ./vendor/bin/phpstan
+PHPSTAN  = ./vendor/bin/phpstan --no-progress
```
Comment (nick-zh) [inline Makefile]: sometimes --no-progress is in here sometimes in the make target, chose one way to do it

File: docker/dev/php/Dockerfile [hunk 3]
```diff
@@ -1,29 +1,32 @@
-FROM php:7.4.0-cli-alpine3.10
+FROM php:7.3-cli-alpine3.10
+
+ARG HOST_USER_ID
 
 # SYS: Install required packages
 RUN apk --no-cache upgrade && \
-    apk --no-cache add bash git sudo make
+    apk --no-cache add bash git sudo autoconf gcc g++ make gettext make shadow
```
Comment (nick-zh) [inline docker/dev/php/Dockerfile]:
```suggestion
    apk --no-cache add bash git sudo autoconf gcc g++ make make shadow
```

File: Makefile (hunk 1 above)
Comment (choks87) [reply Makefile]: ups

File: Makefile (hunk 2 above)
Comment (choks87) [reply Makefile]: This is one place for no progress and applies to all

**Pull request merged**